Kernel/Bus/USB/xHCI/xHCIController.cpp

```diff
@@ -869,34 +869,42 @@ ErrorOr<Vector<xHCIController::TransferRequestBlock>> xHCIController::prepare_no
 {
     auto const& device = transfer.pipe().device();
     auto const slot = device.controller_identifier();
+
     u32 max_burst_payload = 0;
     {
         auto endpoint_id = endpoint_index(transfer.pipe().endpoint_number(), transfer.pipe().direction());
         SpinlockLocker locker(m_slots_state[slot - 1].lock);
         max_burst_payload = m_slots_state[slot - 1].endpoint_rings[endpoint_id - 1].max_burst_payload;
     }
+
     VERIFY(max_burst_payload > 0);
+
+    Vector<TransferRequestBlock> transfer_request_blocks;
     u32 total_transfer_size = transfer.transfer_data_size();
     auto transfer_request_blocks_count = ceil_div(total_transfer_size, max_burst_payload);
-    Vector<TransferRequestBlock> transfer_request_blocks;
     TRY(transfer_request_blocks.try_resize(transfer_request_blocks_count));
+
     auto offset = 0u;
     for (auto i = 0u; i < transfer_request_blocks_count; ++i) {
         auto& transfer_request_block = transfer_request_blocks[i];
         auto buffer_pointer = transfer.buffer_physical().get() + offset;
-        transfer_request_block.normal.data_buffer_pointer_low = buffer_pointer;
-        transfer_request_block.normal.data_buffer_pointer_high = buffer_pointer >> 32;
```

Comment:
The newlines are fine.

On the LHS, `buffer_pointer` was grouped with its uses, all `trb_transfer_length`-related code was grouped together, and the `transfer_request_block` fields were written in the order of their declaration in the struct.

THe RHS also has a scheme that makes sense, but neither seems obviously better. (…or am I missing something?) So I'd probably undo the line reordering.

Reply:
I undid the reordering. Is this better now?